app/src/sharedTest/java/org/oppia/app/profile/AddProfileActivityTest.kt

```diff
@@ -103,6 +103,7 @@ class AddProfileActivityTest {
     }
   }
 
+  /* ktlint-disable max-line-length */
```

Comment:
This is wrong because what this comment says is disable the ktlint check below for all the below line of code. 
As per the wiki example it will be something like 
`fun testAddProfileActivity_changeConfiguration_inputName_clickCreate_checkOpensProfileChooserActivity() { // ktlint-disable max-line-length`

Same at all places

Reply:
Okay, let me fix that in a minute.

Reply:
@anandwana001 I added this at only 2 places. So, should I change this at only 2 places?

Reply:
Update for your current work only, as this PR handles this particular issue to solve.

Reply:
sure thanks!